test/safe.test.ts

```diff
@@ -34,7 +34,7 @@ async function waitForTx(
   return (await txPromise).wait();
 }
 
-async function swap(trustedNonce: number, trustedAmount: number) {
+async function ticket(trustedNonce: number, trustedAmount: number) {
```

Comment:
We are testing a swap.
```suggestion
async function swap(trustedNonce: number, trustedAmount: number) {
```